models/articles.models.js

```diff
@@ -24,6 +24,7 @@ const fetchArticles = (sort_by = "created_at", order = "desc") => {
     "comments_count",
   ];
   const validOrderValues = ["asc", "desc"];
+  const validTopics = ["mitch", "cats", "paper"];
```

Comment:
We don't want to hardcore this, if we were to add more it would mean we would constantly need to update this array. we can use the DB to know which topics are valid.